OneCommunityGlobal/HighestGoodNetworkApp
# Tianjue fix timelog icon format

# Description
Fix TIME LOG COMPONENT
(PRIORITY MEDIUM) Jae: Fix appearance of icons when logging tangible/intangible time
Log time and you’ll see what’s below for the description field 
<img width="788" alt="Screen Shot 2023-03-23 at 5 38 10 PM" src="https://user-images.githubusercontent.com/70128933/227380226-18d8d5cc-b530-45cb-b9d9-b6c44789390b.png">

## Mainly changes explained:
Adjust the CSS file and block the icon.

## How to test:
1. check into current branch
2. do `npm install` and `...` to run this PR locally
3. login in as any role
4. go to dashboard→ log time→ you will see the icons are formatted

## Note:
It should also work for the icons on the block to write "Weekly summaries".
